CHANGES.md

```diff
@@ -1,3 +1,8 @@
+## v1.0.0 (2024-07-17)
+
+* **breaking change** Replace `Cstruct.t` by `string` (@dinosaure, @hannesm, #167)
+* Remove P224 (@dinosaure, @hannes, #166)
+
```

Comment:
maybe worth to mention that serial is now a string, and enforced to be a positive integer of at most 20 bytes in length.

Reply:
Done :+1:.